hcn/hcnpolicy.go

```diff
@@ -135,6 +139,26 @@ type SDNRoutePolicySetting struct {
 	NeedEncap         bool   `json:",omitempty"`
 }
 
+// A ProxyType is a type of proxy used by the L4 proxy policy.
+type ProxyType int
+
+const (
+	// ProxyTypeVFP specifies a Virtual Filtering Protocol proxy.
+	ProxyTypeVFP ProxyType = iota
+	// ProxyTypeWFP specifies a Windows Filtering Platform proxy.
+	ProxyTypeWFP
+)
+
+// FiveTuple is nested in L4ProxyPolicySetting for WFP support.
+type FiveTuple struct {
```

Comment:
Looks similar to AclPolicySetting, consider reusing.

Reply:
My goal was to match the schema defined in HNS, but I can adapt if needed

Reply:
Whoops, didn't know this was defined in the schema.